src/book.js

```diff
@@ -264,8 +272,11 @@ export class Book {
 
   /**
    * Calls the appropriate builder based on [this.format]
+   * Calls the appropriate builder based on `this.format` to generate PDF & populate Previewer.
    *  to generate PDF & populate Previewer
+   * @param {boolean} isPreview - If true, generates only preview content; otherwise, generates both preview content and a downloadable zip.
    * @param isPreview - if it's true we only generate preview content, if it's not true... we still
```

Comment:
looks like the info in this doc comment is doubled-up?

Reply:
fixed